recipes/aiopeewee/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "aiopeewee" %}
+{% set version = "0.4.2" %}
+{% set sha256 = "be9c5f574dd353b1869fa61982840c753c50ffc0538b6a9f3625bb4990e46fa8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]
```

Comment:
Does it not work for python 3.5? This isn't documented in the package as far as I can tell.

Reply:
No, aiopeewee uses async generators too which requires 3.6